conda.recipe/bld.bat

```diff
@@ -1,5 +1,7 @@
-cd %RECIPE_DIR%\..
-"%PYTHON%" setup.py install
-
-if errorlevel 1 exit 1
+@echo off
 
+SET BLD_DIR=%CD%
+cd /D "%RECIPE_DIR%\.."
+FOR /F "delims=" %%i IN ('git describe --tags') DO set BLAZE_VERSION=%%i
```

Comment:
`BLAZE_VERSION` looks like it was copied over from some other project.  Otherwise this seems fine.  Have you tested it with `conda build`?